features/logging/4.x/kibana.feature

```diff
@@ -0,0 +1,52 @@
+@clusterlogging
+@commonlogging
+Feature: Kibana related features
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-25599
+  @admin
+  @destructive
+  Scenario: Show logs on Kibana web console according to different user role
+    Given I switch to the first user
+    Given I create a project with non-leading digit name
+    Given evaluation of `project` is stored in the :proj clipboard
+    When I run the :new_app client command with:
+      | file | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/loggen/container_json_unicode_log_template.json |
+    Then the step should succeed
+    And I wait up to 600 seconds for the steps to pass:
+    """
+    And a pod becomes ready with labels:
+      | run=centos-logtest,test=centos-logtest |
+    """
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-logging" project
+    And I wait for the "kibana" route to appear
+    And I wait for the "project.<%= cb.proj.name %>" index to appear in the ES pod with labels "es-node-master=true"
+    Given I switch to the first user
+    And I open kibana console in a browser
+    And I perform the :kibana_find_index_pattern web action with:
+      | index_pattern_name | project.<%= cb.proj.name %>.<%= cb.proj.uid %>.* |
+    Then the step should succeed
+    Given cluster role "cluster-admin" is added to the "first" user
```

Comment:
Could you use step `the first user is cluster-admin`?

Reply:
I have a question: what are the differences between step `cluster role "cluster-admin" is added to the "first" user` and step `the first user is cluster-admin` ?

Reply:
No difference. Just slightly different semantics. But given later comment, I think keeping your original step makes more sense.